core/test/user-flow-test.js

```diff
@@ -32,6 +19,11 @@ await td.replaceEsm('../gather/timespan-runner.js', timespanModule);
 
 const mockRunner = await mockRunnerModule();
 
+// Some imports needs to be done dynamically, so that their dependencies will be mocked.
+// See: https://jestjs.io/docs/ecmascript-modules#differences-between-esm-and-commonjs
+//      https://github.com/facebook/jest/issues/10025
```

Comment:
seems weird to still be linking to jest docs for this, though

Reply:
It matches everything else so probably good for now